sys/net/gnrc/network_layer/ipv6/nib/_nib-arsm.c

```diff
@@ -281,7 +281,7 @@ void _handle_snd_ns(_nib_onl_entry_t *nbr)
                 _set_nud_state(netif, nbr,
                                GNRC_IPV6_NIB_NC_INFO_NUD_STATE_UNREACHABLE);
             }
-            /* falls through intentionally */
+            /* intentionally falls through */
```

Comment:
Dito (also has GCC become so picky??!!? oO)

Reply:
Don't know if we have a stricter fall through level now, but assuming we have `level 3`[1],
then the previous string doesn't match the given regex. However, my compiler was able to compile it sometimes, and other times it didn't want to .. with the provided change it worked every time so far.

[1] https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html